src/test/kotlin/com/exactpro/th2/http/client/CommonTests.kt

```diff
@@ -42,8 +43,7 @@ class CommonTests {
         val rawMessage = request.toProtoMessage(ConnectionID.getDefaultInstance(), 12345L)
         val newMetadata = rawMessage.metadata.propertiesMap
         assertEquals(metadata.values.size + 2 /* method and uri */ + 1 /* th2-request-id */, newMetadata.values.size)
-        assertContains(newMetadata, "th2-request-id")
-        assertFalse(newMetadata["th2-request-id"]?.isBlank() ?: true)
+        assertFalse(newMetadata[TH2_REQUEST_ID_PROPERTY].isNullOrBlank(), "The $TH2_REQUEST_ID_PROPERTY message property is null or blank")
```

Comment:
Usually, it is better not to use constants in the test if they are exposed to the user. When you use the same constant in the app code and in the tests, you can change it without breaking the tests. That might cause problems because we break the API for the user.
But in our case, I believe you can ignore that comment.